keps/2724-topology-aware-scheduling/kep.yaml

```diff
@@ -31,6 +31,7 @@ milestone:
 # List the feature gate name and the components for which it must be enabled
 feature-gates:
   - name: TopologyAwareScheduling
+  - name: LeastAllocatedTAS
```

Comment:
```suggestion
  - name: TASLeastAllocated
```
nit, maybe we have more feature gates like this, and prefixing them with TAS will make them sort nicely.

Reply:
Done

Reply:
This seems not committed. PTAL